web/init/src/components/config_render/FileInput.jsx

```diff
@@ -46,32 +55,43 @@ export default class FileInput extends React.Component {
   }
 
   render() {
-
     let label;
-    if (this.props.label) {
-      label = this.props.label;
-    } else if (this.props.multiple) {
-      label = "Choose files";
-    } else {
-      label = "Choose file";
-    }
+    this.props.label ? label = this.props.label : this.props.multiple
+      ? label = "Upload files" : label = "Upload a file";
+
 
     return (
-      <span>
-        <span className={`${this.props.readonly ? "readonly" : ""} ${this.props.disabled ? "disabled" : ""}`}>
-          <p className="sub-header-color field-section-sub-header u-marginTop--small u-marginBottom--small">{label}</p>
-          <input
-            ref={(file) => this.file = file}
-            type="file"
-            name={this.props.name}
-            className="form-control"
-            onChange={this.handleOnChange}
-            readOnly={this.props.readOnly}
-            multiple={this.props.multiple}
-            disabled={this.props.disabled} />
-        </span>
+      <div>
+        <div className={`${this.props.readonly ? "readonly" : ""} ${this.props.disabled ? "disabled" : ""}`}>
+          <p className="sub-header-color field-section-sub-header u-marginTop--small u-marginBottom--small u-marginTop--15">{label}</p>
+          <div className="flex flex-row">
+            <div className={`${this.state.fileAdded ? "file-uploaded" : "custom-file-upload"}`}>
+              <input
+                ref={(file) => this.file = file}
+                type="file"
+                name={this.props.name}
+                className="inputfile"
+                id={`${this.props.name} "selector"`}
```

Comment:
Does `selector` need to be in quotes here? (and also on the label below it)

Reply:
doesn't need to be, fixing!

Reply:
👍